src/test/java/uk/gov/ukho/apitest/tests/ExampleApiTest.java

```diff
@@ -20,10 +23,48 @@ public void setup() {
 
     @Test
     public void WhenCallPostsEndpointThenSuccessStatusReturned() {
-
         response = exampleApiService.getAllPosts();
 
         assertThat(response.statusCode())
+                .as("status code")
                 .isEqualTo(HttpStatus.SC_OK);
     }
+
+    @Test
+    public void WhenCallPostsEndpointThenJsonBodyIsCorrect() {
+        response = exampleApiService.getAllPosts();
+
+        JsonPath jsonPath = response.jsonPath();
+
+        SoftAssertions softly = new SoftAssertions();
+
+        softly.assertThat(jsonPath.getString("id"))
+                .as("id content")
+                .contains("1", "2");
+
+        softly.assertThat((int) jsonPath.get("id.size"))
+                .as("id size")
+                .isEqualTo(100);
+
+        softly.assertAll();
```

Comment:
You could add a JUnit rule then you wouldn't have to call softly.assertAll()

Reply:
Not that it's a useful comment, but I think we had something similar to Softly inside the Tides tests somewhere, worth looking how we did those?

Reply:
Couldn't see it.
On reflection, I think the current approach is the most easily understood and as this is a project to be used by those new to Java, that's a prime consideration.